PencilCase/PencilCase/build.json

```diff
@@ -1,11 +1,11 @@
 {
      "android": {
-         "release": {
-             "keystore": "",
-             "storePassword": "",
-             "alias": "",
-             "password" : "",
-             "keystoreType": ""
-         }
+      "release": {
+        "keystore": "c:\\dev\\jd-release-key.keystore",
```

Comment:
Need a private key, and configure the build.json file.
Refer to issue description for more details.